app/views/admin/shared/_logo_upload.html.erb

```diff
@@ -2,7 +2,7 @@
   <%= f.label :logo, class: "crayons-field__label" do %>
     Logo
   <% end %>
-  <p id="logo-subtitle" class="crayons-field__description">Your logo will display in the upper left hand corner of your Forem. Upload a PNG or JPEG file.</p>
+  <p id="logo-subtitle" class="crayons-field__description">Your logo will display in the upper left corner of your Forem. Upload a PNG or JPEG with a maximum size of 4096x4096px.</p>
```

Comment:
Are those dimensions defined somewhere in the application code?  It would be nice to synchronize that.

In looking at https://github.com/DarthSim/carrierwave-bombshelter, there's an instance method for [max_pixel_dimensions](https://github.com/DarthSim/carrierwave-bombshelter/blob/master/lib/carrierwave/bombshelter.rb#L30-L32).  Do we have an uploader instance available to ask it for it's constraint?

Reply:
These dimensions are the [default dimensions set by carrierwave](https://github.com/forem/forem/blob/main/app/uploaders/base_uploader.rb#L2), which we are relying on and using as our default, maximum dimensions.